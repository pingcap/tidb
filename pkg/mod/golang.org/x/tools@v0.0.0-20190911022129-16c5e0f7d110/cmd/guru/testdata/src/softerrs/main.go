package main

// Tests of various queries on a program containing only "soft" errors.
// See go.tools/guru/guru_test.go for explanation.
// See main.golden for expected query results.

func _() {
	var i int // "unused var" is a soft error
}

func f() {} // @callers softerrs-callers-f "f"

func main() {
	f() // @describe softerrs-describe-f "f"
}
