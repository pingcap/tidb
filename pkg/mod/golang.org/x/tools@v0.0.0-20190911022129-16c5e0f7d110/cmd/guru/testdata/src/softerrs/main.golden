-------- @callers softerrs-callers-f --------
softerrs.f is called from these 1 sites:
	static function call from softerrs.main

-------- @describe softerrs-describe-f --------
reference to func f()
defined here

